.gitignore

```diff
@@ -6,6 +6,8 @@ dist
 npm-debug.log
 
 bundle*.js
+model/version.go
```

Comment:
I looked at -ldflag originally, but after googling around it sounds like they changed how that flag works form go 1.4 and go 1.5.  It doesn't sound like you can use it for both.